<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TProxy</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QMenuBar" name="menubar">
     <widget class="QMenu" name="menuUtils">
      <property name="title">
       <string>Utils</string>
      </property>
      <addaction name="actionConnectivity_check"/>
     </widget>
     <widget class="QMenu" name="menuHelp">
      <property name="title">
       <string>Help</string>
      </property>
      <addaction name="actionMay_Mohsen_helps_you"/>
     </widget>
     <addaction name="menuUtils"/>
     <addaction name="menuHelp"/>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout" rowstretch="0,10,0,0,1">
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="label_tproxynodes">
         <property name="text">
          <string>TProxy nodes:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" rowspan="2" colspan="4">
        <widget class="QListWidget" name="listWidget_nodes">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="iconSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QLabel" name="label_log">
         <property name="text">
          <string>Log</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="pushButton_removenode">
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="pushButton_addnode">
         <property name="text">
          <string>+</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QPushButton" name="pushButton_disconnect">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Disconnect</string>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QPushButton" name="pushButton_connect">
         <property name="text">
          <string>Connect</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="4">
        <widget class="QTextEdit" name="textEdit_log">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
  <widget class="QStatusBar" name="statusbar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>3</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <action name="actionConnectivity_check">
   <property name="text">
    <string>Connectivity check</string>
   </property>
  </action>
  <action name="actionMay_Mohsen_helps_you">
   <property name="text">
    <string>May Mohsen helps you</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_connect</sender>
   <signal>clicked(bool)</signal>
   <receiver>pushButton_connect</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>399</y>
    </hint>
    <hint type="destinationlabel">
     <x>248</x>
     <y>399</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_connect</sender>
   <signal>clicked(bool)</signal>
   <receiver>pushButton_disconnect</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>399</y>
    </hint>
    <hint type="destinationlabel">
     <x>160</x>
     <y>399</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_disconnect</sender>
   <signal>clicked(bool)</signal>
   <receiver>pushButton_connect</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>160</x>
     <y>399</y>
    </hint>
    <hint type="destinationlabel">
     <x>248</x>
     <y>399</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_disconnect</sender>
   <signal>clicked(bool)</signal>
   <receiver>pushButton_disconnect</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>160</x>
     <y>399</y>
    </hint>
    <hint type="destinationlabel">
     <x>160</x>
     <y>399</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_connect</sender>
   <signal>clicked(bool)</signal>
   <receiver>listWidget_nodes</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>285</x>
     <y>402</y>
    </hint>
    <hint type="destinationlabel">
     <x>268</x>
     <y>354</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_disconnect</sender>
   <signal>clicked(bool)</signal>
   <receiver>listWidget_nodes</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>121</x>
     <y>406</y>
    </hint>
    <hint type="destinationlabel">
     <x>108</x>
     <y>349</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
